inference_rules.adoc

```diff
@@ -428,6 +428,81 @@ division allows calibration for quantization and does not allow any retraining.
 The unqualified name “MLPerf” must be used when referring to a Closed Division
 suite result, e.g. “a MLPerf result of 4.5.”
 
+=== Network Division
+
+The Network division inherits all requirements from the Closed division and imposes further constraints.
+In the Network division the SUT is connected to the Loadgen system over a network fabric. The Query Dispatch Library (QDL) component is a submitter-implemented SUT proxy that runs on the Loadgen system. The Network division supports only the Datacenter suite.
+
+==== QDL Constraints
+
+* The QDL is not allowed to do any pre-processing.
+* If an SUT compresses its output, the QDL must decompress the output. Decompression is a timed operation. No other post-processing in the QDL is allowed.
+* The QDL is not allowed to batch queries.
+* The Name method's return value must contain the substring "Network SUT".
+* The Name method's implementation must include at least one round trip over the network. The Name method must not return until the round trip is complete.
+
+==== General Constraints
+
+MLPerf distinguishes between fabric interconnects and bus interconnects. Fabric interconnects are required and bus interconnects are forbidden.
+
+A fabric interconnect must:
+
+1. Work as out-of-the-box chassis-to-chassis interconnect
+1. Use wireless, copper, or fiber-optics media
+1. Suitable for connecting systems above 10 meters distance
+1. Use switch topology
+1. Be highly scalable, reliable, and fault-tolerant
+
+Currently permitted fabric interconnects are Ethernet, IEEE802.11, Infiniband, and 3GPP.
+
+Examples of forbidden bus interconnect include: PCIe/CXL/CCIX, Hypertransport, NVLink, QPI, UPI, and ICI (interchip interconnect).
+
+Additionally, any interconnect not listed in the permitted list is forbidden unless clearance is first obtained from the MLPerf Inference WG.
+
+The SUT and QDL must run on two physically separated and distinct systems.
+
+Fabric and protocol must be reported in the submission metadata. Submission metadata must be sufficient to determine OSI layers one through four of the submission's network stack.
```

Comment:
@liorkhe Could you also add the relevant fields for Loadgen-over-network to the submission system JSON table in submission_rules.adoc and create another PR? This comment is not a blocker for this PR.

Reply:
I created a PR - https://github.com/mlcommons/policies/pull/110